crm_compassion/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_event_compassion_manager","Full access on event_compassion","model_crm_event_compassion","base.group_sale_manager",1,1,1,1
 "access_event_compassion_salesman","Access on event_compassion","model_crm_event_compassion","base.group_sale_salesman_all_leads",1,1,1,1
+"access_demand_planning","Full access on demand_planning","model_demand_planning","base.group_erp_manager",1,1,1,1
```

Comment:
Add also a readonly rule for group  `child_compassion.group_sponsorship`
Add also security accesses for model_demand_weekly_demand